README.md

```diff
@@ -1,101 +1,39 @@
 # TypeSpec
 
+[Official Docs](https://typespec.io/)
```

Comment:
Did you mean to have those split per line? it looks a little crammed now 
<img width="680" alt="image" src="https://github.com/microsoft/typespec/assets/1031227/9cad7cf0-49c8-48f3-b6ff-63b43b7b1e44">
Maybe separated with a ` | `

Reply:
Fixed. My preview app was showing them on different lines. I did what you suggested